bundler/spec/install/gems/fund_spec.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+RSpec.describe "bundle install" do
+  context "with gem sources" do
+    context "when gems include a fund URI" do
+      it "should display the plural fund message after installing" do
+        gemfile <<-G
+          source "#{file_uri_for(gem_repo1)}"
+          gem 'has_metadata'
+          gem 'has_funding'
+          gem 'rack-obama'
+        G
+
+        bundle :install
```

Comment:
Unless you want to be explicit for a readabilty reason (maybe in the last spec in this file it makes sense to keep them separate), use `install_gemfile` over `gemfile` +  `bundle :install`.